labgrid/remote/common.py

```diff
@@ -39,12 +39,25 @@ def extra(self):
         """extra resource information"""
         return self.data['params'].get('extra', {})
 
+    @property
+    def proxy(self):
```

Comment:
Why did we decide to add these as top-level members instead of using the existing extra dict?
I remember we discussed this, but don't remember the actual reason.

Reply:
Because the `extra` attribute is only attached to the ResourceEntry and we need use the proxy information after creating the resource.  We need the Resource information attached to the Resource to later decide whether we need to create the tunnel or not.